### DISCUSSION TITLE
does Awakened Poe trade uses POE Trade Api?

### DISCUSSION THREAD

**Discussion by Norby93** [SnosMe/awakened-poe-trade#866] [Q&A]
does Awakened Poe trade use POE Trade Api? or it is some other sort for searching items?

**Comment by SnosMe** [comment]
Yes, the same one the official site uses.

**Comment by Norby93** [comment]
Спасибо за ответ.

он публичный? я просто хочу сам софтину написать, но так как я только начинаю, мне сложно понять поскольку на официальном ничего не сказано про трейд. https://www.pathofexile.com/developer/docs/reference
ЗЫ извини за нубские вопросы.

**Reply by SnosMe** [reply]
publicly accessible, but not an offical api

**Reply by SnosMe** [reply]
use Network tab in devtools and do some item search to find how it works

**Reply by Norby93** [reply]
> publicly accessible, but not an offical api

what doet it mean?) https://www.pathofexile.com/api/trade/search/Sanctum are you about this?
do i need to register my app by ggg (oauth2) to use api?

> use Network tab in devtools and do some item search to find how it works

got ya

**Reply by technowizzard** [reply]
> > publicly accessible, but not an offical api
> 
> what doet it mean?) https://www.pathofexile.com/api/trade/search/Sanctum are you about this? do i need to register my app by ggg (oauth2) to use api?
> 
> > use Network tab in devtools and do some item search to find how it works
> 
> got ya

What is the result ? Do I need to authorize the app ?
